### DISCUSSION TITLE
GoPro motion detection observations

### DISCUSSION THREAD

**Discussion by suyunas** [gopro/labs#129] [Q&A] [answered]
I am using GoPro 10/11 and for motion detection. Observations:

1.	Without a mask, motion detection triggers the recording and stops when there is no motion across the entire frame
2.	With a mask, motion detection inside the masked region triggers the recording (no recording is triggered by motion outside the masked region), and stops when there is no motion across the entire frame (ie the mask region is ignored when checking for no motion)

Interpretations:
1.	There is a bug which prevents the recording being stopped when there is only motion outside of the masked region (ie even though the masked region is static, the recording is not stopped)
OR
2.	The mask is only used to trigger the recording and not stop it

**Comment by dnewman-gpsw** [comment]
1. Sound like you had the sensitivity set too high or there is movement in the corners.  It doesn't fail in my testing.
2. This is known, fix is coming.

**Reply by suyunas** [reply]
Hi, I have tested my GoPro 10/11 both with the following settings:
Resolution: 1080P,Frames: 30,Lens: linear,Hyper smooth: off, Scheduled capture: off, Duration: no limit, 
Hindsight: off, Timer: off, Zoom: 1.0x, Bit rate: standard, Shutter speed: 1/480, White balance: auto, ISO min: 100, ISO max: 800, Sharpness: medium, Color: natural, Raw audio: off, Wind: off

For motion detection the settings are as follows: 
![image](https://user-images.githubusercontent.com/81650263/198131576-ffa1cc88-4bd1-4cb6-ba80-fc3bc7d0a9fc.png)

Would you like to share your settings to align the process? Many thanks

**Reply by dnewman-gpsw** [reply]
That exact setup works fine on both HERO10 and 11 when I tried it here.  Note: the forcing a 1/480 shutter, certain lighting conditions may cause more noise, and make triggering more sensitive.  

Start simple.  Factory Reset your camera (!FRESET command) And start with defaults are learn what sensitivity you need for the lighting environment.

**Comment by suyunas** [comment, marked as answer]
Fix:  Prepare mask using higher frame rate i.e, 1080P60 and then test it on lower frame rates i.e, 1080P30. Shutter speed or sensitivity are having no effect on stop recording.
